pages/index.js

```diff
@@ -3,63 +3,117 @@ import styles from '../styles/Home.module.css'
 
 export default function Home() {
   return (
-    <div className={styles.container}>
+    <div>
       <Head>
-        <title>Create Next App</title>
+        <title>Alpha Site</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <main className={styles.main}>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
+      <header>
+        <figure>
+          <img src="#" alt="logo.jpg"></img>
+          <figcaption>
+            Governement of Canada
+            Gouvernement du Canada
+          </figcaption>
+        </figure>
```

Comment:
I would say double-check what Canada.ca uses for their header. We probably don't want a figure here, they are using `<div>`s on Canada.ca

Eg: https://design.canada.ca/mandatory-templates/home-page.html#specifications
"Working examples" links to the main canada.ca homepage: https://www.canada.ca/en.html

For the header, it looks like they are using 

```
<div>
    <a href="#" >
	<img src="..." alt="Government of Canada" property="logo">
        <span lang="fr">Government of Canada</span>
</div>
```